week2/index.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+const fs = require("fs");
+
+const todoFile = "todos.txt";
+const helpFile = "help.txt";
+
+const command = process.argv.slice(2);
+let tasks = "";
+
+
+function readTodoFile() {
+
+    fs.readFile(todoFile, "utf8", (err, data) => {
+
+        if (err) {
+            handleError(todoFile, err);
+        }
+        tasks = tasks.concat(data);
+
+        checkCommand();
+    });
+}  
+
+
+function writeTodoFile() {
+
+    fs.writeFile(todoFile, tasks, (err) => {
+
+        if (err) {
+            handleError(todoFile, err);
+        }
+        console.log("Successfully made changes to list ");
+    });  
+}
+
+
+function displayHelp() {
+
+    fs.readFile(helpFile, "utf8", (err, data) => {
+
+        if (err) {
+            handleError(helpFile, err);
+        }
+        console.log(data);
+    });
+}
+
+
+
+function handleError(file, err) {
+
+    if (err.code === "ENOENT") {
+        console.log(`${file} does not exist`);
+
+        if (file === todoFile) {
+            console.log(`Create file ${todoFile} to begin`);
+        }
+    }
+    else {
+        console.log(`An unknown error occurred: ${err.message}`);
+    } 
+    process.exit();
+}
+
+
+function checkCommand() {
+
+    let taskList = tasks.split("\n");
+    const taskNumber = Number(command[1]);
+    const taskIndex = taskNumber - 1;
+
+    switch (command[0]) {
+
+        case "list":
+            !tasks ? console.log("There are no items on the list") : console.log(tasks);
+            break;
+
+        case "add":
+            if (!command[1] || command.length !== 2 || typeof command[1] !== "string") {
+                displayHelp();
+            }
+            else {
+                tasks = tasks.concat(`${command[1]}\n`);
```

Comment:
Here you use `command[1]` as if it's the string contents of the task you want to add, but on line 70, you use `command[1]` as a number - how is that?